omero/sysadmins/whatsnew.txt

```diff
@@ -1,14 +1,11 @@
 What's new for OMERO 5.1
 ========================
 
-From a sysadmin's point of view, here is some introductory / overview
-text.
-
-- :doc:`server-upgrade` has **not yet** been updated to explain the
-  server upgrade process from OMERO 5.0 to OMERO 5.1.
-
-- :doc:`system-requirements` has been updated to reflect **various
-  things yet to be finalized**.
+- :doc:`version-requirements` provides extensive details about which operating
+  systems and dependency versions we will be supporting for the life of 5.1
+  and the likely changes to these for the next major release (currently slated
+  to be 5.2). :doc:`system-requirements` has been updated to reflect these
+  updates.
```

Comment:
Some discrepancies are inevitable given that an OS release aggregates many different packages with particular versions, but I'm not sure what the problem is here.  Both of these Ubuntu LTS releases provide both java 7 and python 2.7.  But we recommend the newer release over the older release.